src/webots/nodes/WbGeometry.cpp

```diff
@@ -44,7 +44,7 @@
 // Constant used to scale down the line scale property
 const float WbGeometry::LINE_SCALE_FACTOR = 250.0f;
 
-const int gMaxIndexNumberToCastShadows = (2 << 14) - 1;  // 2^15 - 1 (16-bit resolution)
```

Comment:
Would not be better to use `(1 << 16) - 1` instead of `(2 << 15) - 1` ? imo, it match more closely to the `16` in `2^16 -1`

Reply:
I agree  `(1 << 16) - 1` is better than `(2 << 15) - 1`. However `(uint16_t)(-1)` looks a little bit too hacky...